backend/src/schema/resolvers/users/blockedUsers.spec.js

```diff
@@ -0,0 +1,393 @@
+import { createTestClient } from 'apollo-server-testing'
+import createServer from '../../../server'
+import Factory from '../../../seed/factories'
+import { gql } from '../../../jest/helpers'
+import { neode, getDriver } from '../../../bootstrap/neo4j'
+
+const driver = getDriver()
+const factory = Factory()
+const instance = neode()
+
+let currentUser
+let blockedUser
+let authenticatedUser
+let server
+
+beforeEach(() => {
+  authenticatedUser = undefined
+  ;({ server } = createServer({
+    context: () => {
+      return {
+        user: authenticatedUser,
+        driver,
+        neode: instance,
+        cypherParams: {
+          currentUserId: authenticatedUser ? authenticatedUser.id : null,
+        },
+      }
+    },
+  }))
+})
+
+afterEach(async () => {
+  await factory.cleanDatabase()
+})
+
+describe('blockedUsers', () => {
+  let blockedUserQuery
+  beforeEach(() => {
+    blockedUserQuery = gql`
+      query {
+        blockedUsers {
+          id
+          name
+          isBlocked
+        }
+      }
+    `
+  })
+
+  it('throws permission error', async () => {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-14T10:50:38Z" title="Wednesday, August 14th 2019, 12:50:38 pm +02:00">Aug 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,393 @@
+import { createTestClient } from 'apollo-server-testing'
+import createServer from '../../../server'
+import Factory from '../../../seed/factories'
+import { gql } from '../../../jest/helpers'
+import { neode, getDriver } from '../../../bootstrap/neo4j'
+
+const driver = getDriver()
+const factory = Factory()
+const instance = neode()
+
+let currentUser
+let blockedUser
+let authenticatedUser
+let server
+
+beforeEach(() => {
+  authenticatedUser = undefined
+  ;({ server } = createServer({
+    context: () => {
+      return {
+        user: authenticatedUser,
+        driver,
+        neode: instance,
+        cypherParams: {
+          currentUserId: authenticatedUser ? authenticatedUser.id : null,
+        },
+      }
+    },
+  }))
+})
+
+afterEach(async () => {
+  await factory.cleanDatabase()
+})
+
+describe('blockedUsers', () => {
+  let blockedUserQuery
+  beforeEach(() => {
+    blockedUserQuery = gql`
+      query {
+        blockedUsers {
+          id
+          name
+          isBlocked
+        }
+      }
+    `
+  })
+
+  it('throws permission error', async () => {
```


this is describing `unauthenticated`, correct?